.github/ISSUE_TEMPLATE/task.md

```diff
@@ -0,0 +1,14 @@
+---
+name: Task (Internal Use)
+about: This issue is used to track a smaller feature/task that does not require
+  being broken down to milestones. This is used by the core Metabase team to
+  project plan and manage work.
+labels: ".Task"
+
+---
+
+**Links**
```

Comment:
I'm not sure we will have product docs / eng docs for issues. Most of the time we will link to other epics

Reply:
yeah, i think i've only seen a handful. i'll just leave Links as a header then?

Reply:
Changed the header to “Context” but left the call-out to related issues in place after all, since I heard from the folks in Success that it's valuable to know when an issue is closed (some customers might be happy to learn that), and if we just create new issues and don't even link the existing related ones, this process breaks